add-users/add-users.js

```diff
@@ -365,8 +370,10 @@ const getServicePoints = async () => {
   const res = await okapiGet('/service-points?limit=2000');
   if (res.json.totalRecords) {
     return res.json.servicepoints;
+  } else {
+    return {servicePoints:[]}
   }
-  throw "Could not retrieve permissions";
+  throw "Could not retrieve service points";
```

Comment:
Couple things: 

1. `{servicePoints:[]}` is an object containing a `servicePoints` key pointing to an empty array. For parity, if the query fails and you don't want to deal with service points, it would be better to return an empty array, `[]`. 
1. Adding the else clause makes the `throw` clause is unreachable, which is fine, but then it should be removed. Should there be a `console.warn('Could not retrieve service points')` line in the `else` block so folks know?